src/clock-weather-card.ts

```diff
@@ -472,6 +477,18 @@ export class ClockWeatherCard extends LitElement {
     return this.getWeather().attributes.humidity ?? null
   }
 
+  private getApparentTemperature (): number | null {
+    if (this.config.apparent_sensor) {
+      const apparentSensor = this.hass.states[this.config.apparent_sensor] as TemperatureSensor | undefined
+      const temp = apparentSensor?.state ? parseFloat(apparentSensor.state) : undefined
+      const unit = apparentSensor?.attributes.unit_of_measurement ?? this.getConfiguredTemperatureUnit()
+      if (temp !== undefined && !isNaN(temp)) {
+        return this.toConfiguredTempWithoutUnit(unit, temp)
+      }
+    }
+    return NaN
```

Comment:
whats the point of returning `NaN` instead of `null`?

Reply:
no idea, will return null